modules/social_features/social_profile/src/Form/SocialProfileSettingsForm.php

```diff
@@ -193,4 +194,17 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     parent::submitForm($form, $form_state);
   }
 
+  /**
+   * Redirects users from /edit-profile to edit profile page of current user.
+   *
+   * @return \Symfony\Component\HttpFoundation\RedirectResponse
+   *   Returns a redirect to the edit profile page of current user.
+   */
+  public function editProfileRedirect(): RedirectResponse {
```

Comment:
This needs to be its own controller, or at least based on a class that extend ControllerBase.